app/scripts/controllers/otherResources.js

```diff
@@ -70,6 +70,7 @@ angular.module('openshiftConsole')
       if (!selected) {
         return;
       }
+      $scope.selectedResource = APIService.kindToResource(selected.kind);
```

Comment:
I think you are going to need the group as well to correctly check things that are not in one of the base APIs.  Have you checked this with HPAs?